torch/csrc/distributed/c10d/init.cpp

```diff
@@ -38,6 +38,8 @@ namespace {
 constexpr char* GLOO_SOCKET_IFNAME_ENV = "GLOO_SOCKET_IFNAME";
 #endif
 
+constexpr int DEFAULT_BUCKET_BYTES_CAP = int(25 * 1024 * 1024);
```

Comment:
The default value now lives in both Python and C++, do we need to consolidate them?